tests/lib/cli-engine/file-enumerator.js

```diff
@@ -457,13 +457,11 @@ describe("FileEnumerator", () => {
                 });
 
                 it("should throw an error if no files match a glob", () => {
-
-                    // Relying here on the .eslintignore from the repo root
                     const patterns = ["tests/fixtures/glob-util/ignored/**/*.js"];
```

Comment:
```suggestion
                    const patterns = ["dir-does-not-exist/**/*.js"];
```

I think the change in this test case is generally fine because, according to its title, it was supposed to test a scenario where no files match a glob rather than when all matched files are ignored (for which there's already another test case), but this pattern looks confusing now when the `cwd` is `getFixturePath("ignored-paths")`.

Reply:
Updated.